assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -85,6 +89,7 @@ function useInsertElement() {
    */
   const insertElement = useCallback(
     (type, props) => {
+      setZoomSetting(ZOOM_SETTING.FIT);
```

Comment:
Note that this is only required for the text presets, as they cannot be dragged in. Any element that can be dragged in will actually reset the canvas zoom in the `dragStart` handler on the draggable element also for a click.